README.md

```diff
@@ -20,6 +20,66 @@ Fork this repository and add your tables here:
 
 (your tables go here - you can use TableGenerator to create them. Include the data from the sample above in your tables)
 
+### Authors
+
+| AuthorID | FullName      | Email                       | Bio            |
+|----------|---------------|-----------------------------|----------------|
+| 1        | Eric Fleming  | ericfleming@nimblepros.com  | Long Text Here |
+| 2        | Kyle McMaster | kylemcmaster@nimblepros.com | Long Text Here |
+| 3        | Steve Smith   | steve@nimblepros.com        | Long Text Here |
+
+### BlogPosts
+
+| Slug                          | Title                                | Body           | AuthorID | Views |
+|-------------------------------|--------------------------------------|----------------|----------|-------|
+| serilog-structured-logging    | Structured Logging with Serilog      | Long Text Here | 1        | 123   |
+| persisting-smart-enum         | Persisting a Smart Enum with EF Core | Long Text Here | 2        | 234   |
+| localization-aspnet-core-apis | Localization in ASP.NET Core APIs    | Long Text Here | 3        | 345   |
```

Comment:
Looks good! There's not ID column, though. So I assume you're planning on using Slug as the PK. I would recommend using an integer id column since it's shorter and also provides you with consistency if you want to use a common way to access data in your code. You can still add a unique constraint and/or index on slug if desired.

Reply:
Does being shorter mean also a faster query?

Reply:
Yes but typically not enough that you'd notice it until you have GBs of data.